README.rst

```diff
@@ -94,6 +94,20 @@ Example:
 
    if False: print('there')
 
+Black Command Line Arguments
+============================
+
+``black`` `command line arguments`_ that are currently supported:
+
+.. code:: sh
```

Comment:
Isn't the ReStructuredText directive `.. code-block::`, not `.. code::`?

I've used this earlier in the readme:
```suggestion
.. code-block:: shell
```